Jenkinsfile.buildchain

```
@@ -0,0 +1,121 @@
+@Library('jenkins-pipeline-shared-libraries')_
+
+additionalTimeout = "${env.ADDITIONAL_TIMEOUT?.trim() ?: 120}"
+
+agentLabel = "${env.ADDITIONAL_LABEL?.trim() ? ADDITIONAL_LABEL : 'kie-rhel7 && kie-mem16g'} && !master"
+build_jdk_tool = env.BUILD_JDK_TOOL?.trim() ?: 'kie-jdk11'
+build_maven_tool = env.BUILD_MAVEN_TOOL?.trim() ?: 'kie-maven-3.8.1'
+
+pr_type = env.BUILDCHAIN_PR_TYPE?.trim() ?: 'pr'
+build_chain_additional_arguments = env.BUILDCHAIN_PROJECT ? "-sp=${env.BUILDCHAIN_PROJECT.trim()}" : ''
+
+is_native = env.NATIVE ? true : false
+is_lts = env.LTS ? true : false
```

for now, it is represented by `QUARKUS_BRANCH` env, later we could move back to simple `LTS` env

```suggestion
is_lts = env.QUARKUS_BRANCH ? true : false
```